File: AssetManager/Data/DataClasses/DataClassMapping/ColumnNameAttribute.vb
<AttributeUsage(AttributeTargets.[Property])>
Public Class DataColumnNameAttribute
    Inherits Attribute

    Private _columnName As String
    Public Property ColumnName As String
        Get
            Return _columnName
        End Get
        Set
            _columnName = Value
        End Set
    End Property

    Public Sub New()
        _columnName = String.Empty
    End Sub

    Public Sub New(columnName As String)
        _columnName = columnName
    End Sub
End Class

File: AssetManager/Data/DataClasses/DataClassMapping/ColumnNameAttribute.cs
using System;
namespace AssetManager
{
    [AttributeUsage(AttributeTargets.Property)]
	public class DataColumnNameAttribute : Attribute
	{


		private string _columnName;
		public string ColumnName {
			get { return _columnName; }
			set { _columnName = value; }
		}

		public DataColumnNameAttribute()
		{
			_columnName = string.Empty;
		}

		public DataColumnNameAttribute(string columnName)
		{
			_columnName = columnName;
		}

	}
}
